app/routes/public/index.js

```diff
@@ -12,9 +12,9 @@ export default Route.extend({
     return {
       event   : eventDetails,
       tickets : await eventDetails.query('tickets', {
-        reload: true,
-
-        filter: [
+        reload  : true,
+        include : 'event',
```

Comment:
okay, so the event is already their inside `eventDetails`. Using that you are querying its tickets, and then you are including the event within the ticket. The event is already fetched! You are making a loop.

Reply:
Yeah. Will remove the model fetch loop.  So the change inside ` app/templates/components/public/ticket-list.hbs` is sufficient for the tax-info right?

Reply:
Tax is the issue. Not the event or the tickets. You need to pre-fetch the tax.

Reply:
Oh yeah. That's the reason why it's undefined and not accessible. Got it!